docs/guides/proxy_management.mdx

```diff
@@ -81,6 +84,25 @@ When no `session_id` is provided, our proxy URLs are rotated round-robin.
     </TabItem>
 </Tabs>
 
+### Tiered proxies
+
+When you use HTTP proxies in real world crawling scenarios, you have to decide which type of proxy to use to reach a sweetspot between cost efficiency and reliably avoiding blocking. Some websites may allow crawling with no proxy, on some you may get away with using datacenter proxies, which are cheap but likely to get blocked, and sometimes you need to use expensive residential proxies.
+
+To take the guesswork out of this process, Crawlee allows you to configure multiple tiers of proxy URLs. When crawling, it will automatically pick the left-most tier where it doesn't encounter blocking. If you organize your proxy server URLs in tiers so that the left-most tier contains the cheapest, least reliable ones and each higher tier contains more expensive, more reliable ones, you will get an optimal anti-blocking performance.
```

Comment:
I'm fine with it either way, but I always thought of tiers as top-down layers - so the first one is the "lowest", while the last one is the "top tier" (this is afaik an actual English phrase).

Reply:
yeah agreed

Reply:
okay, so higher index ~ higher tier? I'll probably explain the relation of high/low tier to the index in the array anyway

Reply:
yeah, higher is more expensive and better working, that way it feels relatively understandable :D